### PR TITLE
Change description and add source in metadata

### PR DESCRIPTION
**Pull request by arturo-seijas**
Metadata changes to adapt to discourse as per https://discourse.charmhub.io/t/changes-to-charmhub-io-pages/9366

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -4,12 +4,27 @@ summary: Agent for Jenkins, the OSS automation server.
...
+  from [MicroK8s](https://microk8s.io) to [Charmed Kubernetes](https://ubuntu.com/kubernetes) 
+  and public cloud Kubernetes offerings.
+
+  As such, the charm makes it easy for those looking to take control of their own
+  Agents whilst keeping operations simple, and gives them the freedom to deploy on 
+  the Kubernetes platform of their choice.
+
+  For DevOps or SRE teams this charm will make operating Jenkins Agent simple and
+  straightforward through Juju's clean interface. It will allow easy deployment
+  into multiple environments for testing of changes, and supports scaling out for
```
Comment (yanksyoon) [inline metadata.yaml:26]:
```suggestion
  into multiple environments for testing changes, and supports scaling out for
```
A minor suggestion on wording :)

**Review by yanksyoon — APPROVED** [review]
LGTM!

File: metadata.yaml [hunk 2]
```diff
@@ -4,12 +4,27 @@ summary: Agent for Jenkins, the OSS automation server.
...
 description: |
-  Agent for Jenkins, the leading open source automation server. Jenkins provides hundreds of plugins to support building, deploying and automating any project.
+  A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
+  deploying and managing [Jenkins](https://www.jenkins.io/) Agent on Kubernetes, 
+  configurable to use a Jenkins charm deployed in another Juju model, or to connect 
+  to a standalone Jenkins instance.
+
+  This charm simplifies initial deployment and "day N" operations of Jenkins Agent
+  on Kubernetes. It allows for deployment on many different Kubernetes platforms, 
+  from [MicroK8s](https://microk8s.io) to [Charmed Kubernetes](https://ubuntu.com/kubernetes)
```
Comment (yhaliaw) [inline metadata.yaml:17]: Would the link for Charmed Kubernetes be better if it is https://ubuntu.com/kubernetes/charmed-k8s instead?

**Review by yhaliaw — COMMENTED** [review]
A minor suggestion.

File: metadata.yaml (hunk 2 above)
Comment (arturo-seijas) [reply metadata.yaml]: Indeed

**Review by yanksyoon — APPROVED** [review]
LGTM! Thanks for the changes!

**Review by merkata — APPROVED** [review]

**Pull request merged**